packages/salesforcedx-utils-vscode/src/commands/commandletExecutors.ts

```diff
@@ -159,9 +167,16 @@ export abstract class LibraryCommandletExecutor<T>
       const success = await vscode.window.withProgress(
         {
           title: nls.localize('progress_notification_text', this.executionName),
-          location: vscode.ProgressLocation.Notification
+          location: vscode.ProgressLocation.Notification,
+          cancellable: this.cancellable
         },
-        () => this.run(response)
+        (progress, token) => {
+          token.onCancellationRequested(() => {
```

Comment:
We need to handle the check below whether it's successful or not so that both the cancel message and failure message don't show up, just the former.